corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -42,7 +42,7 @@ hqDefine("data_dictionary/js/data_dictionary", [
 
                 for (let prop of group.properties) {
                     const isGeoCaseProp = (self.geoCaseProp === prop.name);
-                    var propObj = propertyListItem(prop.name, prop.label, false, prop.group, self.name, prop.data_type,
+                    var propObj = propertyListItem(prop.name, prop.label, false, prop.group_obj, self.name, prop.data_type,
```

Comment:
The `propertyListItem` here only requires the group name, the `prop.group_obj` can be replaced with group name itself.

Reply:
Changed